src/lib.ts

```diff
@@ -166,7 +166,8 @@ export { default as SharePreviewTwitter } from './components/ShareImage/SharePre
 
 export { default as TeaserEmbedComment } from './components/TeaserEmbedComment'
 export { SeriesNav } from './components/SeriesNav'
-
+export { default as Tabs } from './components/Tabs'
+export { default as Scroller } from './components/Scroller'
```

Comment:
Tabs are just a special case of a Scroller with particular styling of the child nodes. I first hat it so that if you passed an an item with an {element: () } into Tabs, that it would render that instead of the TabItem. But I abandoned that as it got messy with nested buttons...